src/DataTable/TableRow.jsx

```diff
@@ -26,7 +26,7 @@ TableRow.propTypes = {
   /** props to include on the tr tag (must include id) */
   getRowProps: PropTypes.func.isRequired,
   /** cells in the row */
-  cells: PropTypes.arrayOf(PropTypes.shape()).isRequired,
+  cells: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
```

Comment:
Is `PropTypes.object` more appropriate?

Reply:
I'm pretty sure the linter yells at me every time I use `PropTypes.object`?